src/service/OutputService.js

```diff
@@ -0,0 +1,24 @@
+import { Console } from '@woowacourse/mission-utils';
+import { IOMessage } from '../constants/IOMessage.js';
+import { matchLottoNums } from '../util/matchLottoNums.js';
+import calProfitRate from '../util/calculators/calProfitRate.js';
+import { calTotalPrize } from '../util/calculators/calTotalPrize.js';
+import formatNum from '../util/formatters/formatNum.js';
+
+export class OutputService {
+  static printResults(purchasedLottoInfo, winningLotto, bonusNum) {
+    Console.print(IOMessage.WIN_STATISTICS);
+    const results = matchLottoNums(
+      purchasedLottoInfo.purchasedLottoNumbers,
+      winningLotto.getNumbers(),
+      bonusNum,
+    );
+
+    for (const [key, count] of Object.entries(results)) {
+      Console.print(IOMessage[key](count));
+    }
```

Comment:
for of보다 Object, Array 메소드를 사용하는 것을 추천드려요! [11.1](https://github.com/ParkSB/javascript-style-guide?tab=readme-ov-file#iterators--nope)
이런 식으로 할 수 있을 것 같아요!

```suggestion
    Object.entries(results).forEach(([key, count]) => {
      Console.print(IOMessage[key](count));
    }
```